README.md

```diff
@@ -119,30 +119,30 @@ So, what are you waiting for? Go ahead and start customizing your action!
    npm run all
    ```
 
-   > [!WARNING]
-   >
-   > This step is important! It will run [`ncc`](https://github.com/vercel/ncc)
-   > to build the final JavaScript action code with all dependencies included.
-   > If you do not run this step, your action will not work correctly when it is
-   > used in a workflow. This step also includes the `--license` option for
-   > `ncc`, which will create a license file for all of the production node
-   > modules used in your project.
+> [!WARNING]
+>
+> This step is important! It will run [`ncc`](https://github.com/vercel/ncc) to
+> build the final JavaScript action code with all dependencies included. If you
+> do not run this step, your action will not work correctly when it is used in a
+> workflow. This step also includes the `--license` option for `ncc`, which will
+> create a license file for all of the production node modules used in your
+> project.
 
-1. Commit your changes
+5. Commit your changes
```

Comment:
```suggestion
1. Commit your changes
```

Reply:
[An alert markdown](https://github.com/orgs/community/discussions/16925) didn't work within other elements.
Then we would like to use reorder list.

x-ref: [Discussion comment](https://github.com/orgs/community/discussions/16925#discussioncomment-10195289)